packages/ui/src/views/repo/pull-request/details/components/changes/pull-request-changes-filter.tsx

```diff
@@ -354,24 +346,9 @@ export const PullRequestChangesFilter: React.FC<PullRequestChangesFilterProps> =
                   submitReview?.('approved' as PullReqReviewDecision)
                 }
               }}
-              disabled={isActiveUserPROwner}
-              title={isActiveUserPROwner ? 'Self-approval of pull requests is not permitted.' : undefined}
-              variant="split"
```

Comment:
`variant="split"` has been removed from `Button` component and `ButtonWithOptions` will be used instead.